<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="14" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Установка порогов</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item row="15" column="3">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Установить</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="13" column="0">
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>ГСО-ПГС №1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ГСО-ПГС №3</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="15" column="2">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Блок.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Состояние газоанализатора</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Калибровка</string>
          </property>
         </widget>
        </item>
        <item row="15" column="0">
         <widget class="QComboBox" name="comboBox_2">
          <item>
           <property name="text">
            <string>Порог 1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Порог 2</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="8" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Сетевой адрес</string>
          </property>
         </widget>
        </item>
        <item row="16" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="7" column="0" colspan="4">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2" colspan="2">
         <widget class="QLabel" name="labelThreshold2">
          <property name="text">
           <string>Порог 2: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QLabel" name="labelState">
          <property name="text">
           <string>Статус:</string>
          </property>
         </widget>
        </item>
        <item row="13" column="3">
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>Калибровать</string>
          </property>
         </widget>
        </item>
        <item row="9" column="3">
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Задать</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="pushButtonRefreshData">
          <property name="text">
           <string>Обновить</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0" colspan="3">
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
        <item row="13" column="1" colspan="2">
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="labelAddress">
          <property name="text">
           <string>Адрес:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QLabel" name="labelThreshold1">
          <property name="text">
           <string>Порог 1: </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <widget class="QLabel" name="labelConcentration">
          <property name="text">
           <string>Текущие показания: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditEventLog">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>783</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_COM"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="action_6"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="separator"/>
  </widget>
  <action name="action_3">
   <property name="text">
    <string>фыва</string>
   </property>
  </action>
  <action name="action_COM">
   <property name="text">
    <string>Настройки COM порта</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
